### ISSUE TITLE
PR labels unreadable in dark themes

### ISSUE THREAD

**Issue by Arcanemagus** [atom/github#772]
### Description

The labels associated with a PR use the UI theme's text, but the background color from GitHub, leading to conflicts that can make them illegible.

### Steps to Reproduce

1. Have a branch checked out with an associated PR that has a label
2. Attempt to read the label(s) of the PR

**Expected behavior:**

Labels are legible.

**Actual behavior:**

Since the labels use the theme color for text, if the theme uses a light color and the label has a light background it becomes near unreadable.

Atom Material (v2):
![image](https://cloud.githubusercontent.com/assets/427137/25876969/6521cee4-34d5-11e7-8c0d-9c59410df613.png)

One Dark:
![image](https://cloud.githubusercontent.com/assets/427137/25876981/75538a5a-34d5-11e7-9522-423b2d628e17.png)

One Light: (as a usable example)
![image](https://cloud.githubusercontent.com/assets/427137/25876992/8477f37c-34d5-11e7-93b2-30c9762692dd.png)

The label on github.com:
![image](https://cloud.githubusercontent.com/assets/427137/25877039/cca0ca3e-34d5-11e7-99dc-0ca1ee7a4c7a.png)

**Reproduces how often:**

100% of the time.

### Versions

Windows 10 x64

Build from https://github.com/atom/atom/pull/14372/commits/4cbff3044604988e2c2553af0d78e71c13c25696.

Atom    : 1.18.0-dev-4cbff3044 x64
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0

github@0.0.2 (bundled in the build)

### Additional Information

The PR/Branch in question was https://github.com/AtomLinter/linter-phpcs/pull/261.

**Comment by simurai** [comment]
On .com, the text color gets generated based on what background color a label has:

![screen shot 2017-05-12 at 2 48 00 pm](https://cloud.githubusercontent.com/assets/378023/25983903/7bf6fcb2-3722-11e7-9929-b5fbaaae8bc2.png)

But it seems the [API](https://developer.github.com/v3/issues/labels/) **only** provides the background color. So unless the API can be extended to also provide the text color, we would have to use some contrast function. Maybe TinyColor's  [mostreadable](https://github.com/bgrins/TinyColor#mostreadable):

```js
tinycolor.mostReadable( color, ["#fff", "#000"]).toHexString();
```

☝️  should automatically pick either black or white.

**Comment by DrianHillman** [comment]
Hi! 👋🏽  

I'm new to contributing to this project (but not new to Atom); this looks like a great starting point. I'd be happy to submit an implementation for this!

**Comment by BinaryMuse** [comment]
@DrianHillman That sounds great! Feel free ^__^

**Comment by 50Wliu** [comment]
Fixed by #1000.
